src/components/Tearsheet/Tearsheet.js

```diff
@@ -149,7 +156,10 @@ class Tearsheet extends Component {
                 </section>
               )}
 
-              <section className={`${namespace}__main`}>
+              <section
+                aria-hidden={this.state.loading}
+                className={`${namespace}__main`}
+              >
                 {!closeButton.isDisabled && (
                   <IconButton
```

Comment:
This just occurred to me and is likely scope creep that can be resolved in a follow-up, but should the user be prevented from closing the `Tearsheet` using the close button while it's loading? @petervachon

Reply:
Good point!

Reply:
@petervachon did you have any thoughts about @SimonFinney's question above?

Reply:
Yes, the user should be prevented from closing the tearsheet while it's loading.